### DISCUSSION TITLE
Automatically recognize the use of CSS Modules, and you use them as CSS Modules.

### DISCUSSION THREAD

**Discussion by 521707** [vercel/next.js#16504] [Help]
![image](https://user-images.githubusercontent.com/37111718/90995845-c0094980-e5ef-11ea-96e5-1ff751ef5d1b.png)
As shown in the figure.
Similar ,  https://umijs.org/docs/assets-css#css-modules

// Need. CSS Modules
import styles from './foo.less';

// No  CSS Modules  
import './foo.less';

**Comment by 521707** [comment]
@jamesmosier 
@maranomynet   
help me ~

**Comment by websocket98765** [comment]
For NextJS, it seems you need to rename the file:
```js
import styles from "Foo.module.css";
```
Then use like:
```js
<div className={style.myclass}>
  hello world
</div>
```
